src/seedu/addressbook/data/AddressBook.java

```diff
@@ -89,6 +94,16 @@ public void addPerson(Person toAdd) throws DuplicatePersonException {
     public void addTag(Tag toAdd) throws DuplicateTagException {
         allTags.add(toAdd);
     }
+    
+    /**
+     * Adds a tag to the person if newly added in the address book.
+     *
+     */
+    public void addTagForPerson(Tag tag, Person person){
+        Tagging newTag = new Tagging(true, tag, person);
```

Comment:
representing add or delete using a boolean may be undesirable because it requires another developer to know that true means...? and false means...? Instead, you can use an Enum here to express Added/Deleted. That allows you to extend behaviour in future to include something like "Updated".